src/gatling/bodies/createResourceAccess.json

```diff
@@ -8,5 +8,5 @@
   "attributePermissions": {
     "": ["CREATE", "READ", "UPDATE"]
   },
-  "securityClassification": "Public"
+  "securityClassification": "PUBLIC"
```

Comment:
A `Public` appears in 2 additional places in postman collections. Can you update it as well?

Reply:
Added in two places in postman collections.